CONTRIBUTING.md

```diff
@@ -4,22 +4,24 @@ Thank you for your contribution!
 
 ## Order
 
-Please ensure `icons_by_filename`, `icons_by_file_extension` and `filetypes` are ordered alphabetically, to prevent merge conflicts.
+Please ensure `icons_by_filename`, `icons_by_file_extension`, `icons_by_operating_system`, `icons_by_desktop_environment`,
```

Comment:
Line breaks at 120 char but could be 100 or 80

Reply:
I'm easy either way, although links often end up >80